src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -810,22 +815,24 @@ private void BuildCookieCollectionFromDomainMatches(Uri uri, bool isSecure, int
                 PathList pathList;
                 lock (m_domainTable)
                 {
-                    if (!m_domainTable.TryGetValue(domainAttribute[i], out pathList))
+                    object pathListValue = m_domainTable[domainAttribute[i]];
```

Comment:
There is no need for `pathListValue ` you can just assign directly to `pathList`